app/build.gradle

```diff
@@ -23,8 +23,14 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
-            // proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            minifyEnabled true
```

Comment:
There are a few issues with minification. We want to add the ability for users to submit stack traces in future. Minifying reduces the usability of such stack traces. Of course, we can add some settings to reach a middle ground but that'll take some extra work. Also, the application size is small enough that size is not much of an issue. I would prefer to leave this out of this PR and discuss this later.

Reply:
Makes sense. Done.